core/src/validator.rs

```diff
@@ -165,6 +165,7 @@ pub struct ValidatorConfig {
     pub no_wait_for_vote_to_start_leader: bool,
     pub accounts_shrink_ratio: AccountShrinkThreshold,
     pub wait_to_vote_slot: Option<Slot>,
+    pub observable_vote_acounts: Arc<HashSet<Pubkey>>,
```

Comment:
Bikeshed alert, 
```suggestion
    pub vote_accounts_to_monitor: Arc<HashSet<Pubkey>>,
```
what does it mean these vote accounts are “observable”? To me that sounds like other vote accounts are somehow not observable?

Also about the `Arc`, it does help to have only a single instance if this is referenced in multiple places, but it’s also a slight complication, a pubkey is only 32 bytes, even if you monitor a thousand of them it’s only a few kilobytes. Perhaps it’s worth just copying the hash set if it’s needed in multiple places.

Though with the `Arc`, it makes the hash set immutable, so at least nobody in the future will try to do something like modify it at runtime and expect it to work. If we copy it, you could modify the copy in one place and then be confused about why the other place doesn’t see the new value.

Neither one is clearly better than the other to me, I have no strong opinion on it.